File: python_projects/cash.py
import basicfunc
def main():
    while True:
        change_owed = basicfunc.get_float("Change: ")#make sure that user don't enter negative num
        if change_owed > 0:
            break
    
    change_owed = change_owed * 100 #from dollar to cents
    mini_num  = 0
    index = 0
    coins = [25,10,5,1]
    while change_owed > 0:
        if change_owed >= coins[index]:
            change_owed -= coins[index]
            mini_num +=1
        else:
            index +=1
    
    print(mini_num)            


main()

File: C_projects/cash/cash.c
#include <stdio.h>
#include "../samples.h"//path to the file header
int main(void)
{
    int change_owed;
    //reprompt the user if input negative
    do
    {
        change_owed = get_int("change owed: ");
    } 
    while (change_owed < 0);        
    int coins[] = {25,10,5,1};
    int count = 0;
    int index = 0;
    while(change_owed > 0)
    {
        if(change_owed >= coins[index])
        {
            change_owed -= coins[index];
            count +=1;
        }
        else
        {
            index +=1;
        }
    }

    printf("%i",count);
}